src/Command/FooHelloCommand.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+namespace App\Command;
+
+use Psr\Log\LoggerInterface;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\ArrayInput;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\DependencyInjection\Attribute\TaggedIterator;
+
+/**
+ * FooHelloCommand can be executed by itself, and can accept and execute a chain of other commands, if any.
+ * Identifies the commands that belong to the chain using their tags.
+ */
+class FooHelloCommand extends Command
+{
+    /**
+     * @var bool Indicates whether the FooHelloCommand has been called.
+     */
+    public static $isFooHelloCommandCalled = false;
+
+    /**
+     * @inheritdoc
+     */
+    protected static $defaultName = 'foo:hello';
+
+    /**
+     * @inheritdoc
+     */
+    protected static $defaultDescription = 'Foo Hello Command';
+
+    protected $commands;
+    protected $logger;
```

Comment:
properties are not typed so it's easier to make a mistake by setting wrong value